core/domain/html_validation_service.py

```diff
@@ -21,12 +21,15 @@
 
 import json
 import logging
+from xml.etree import ElementTree as ET
```

Comment:
Typically we import at the stdlib level -- "import xml" and then do xml.etree.ElementTree.fromstring() in code. E.g. notice that we use "import datetime" rather than "from datetime import datetime".